<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RC-Model V3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_main" rowstretch="0,0" columnstretch="4,0">
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget_statusBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_statusBar"/>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_functions" rowstretch="1,0,0,0" columnstretch="1,0,0">
      <item row="1" column="0">
       <widget class="QLabel" name="label_accu">
        <property name="text">
         <string>Akku Ladung:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Emergency</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="button_run">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="label_sendText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QProgressBar" name="progressBar_accu">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="button_emergency">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border: none;
	outline: none;
	background-image: url(&quot;:/images/german/emergency_button_released.png&quot;);
}

QPushButton:pressed 
{
	background-image: url(&quot;:/images/german/emergency_button_pressed.png&quot;);
}

QPushButton:hover
{
	border: none;
	outline: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_gamepad">
       <attribute name="title">
        <string>Gamepad</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_tabGamepad" stretch="3,1">
        <item>
         <widget class="ImageLabel" name="imageLabel_controller" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="table_gamePad">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>400</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>12</number>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>Button A</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Button B</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Button X</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Button Y</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right  Bumper</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Left Bumper</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Directional pad</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Left X-Axis</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Left Y-Axis</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right X-Axis</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Right Y-Axis</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>Trigger</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="4" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="5" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="6" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="7" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="8" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="9" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="10" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="11" column="0">
           <property name="text">
            <string>0</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_virtualGamepad">
       <attribute name="title">
        <string>Virtuelles Gamepad</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,10" columnstretch="4,1" rowminimumheight="0,0">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_reset">
          <property name="text">
           <string>Werte zurücksetzen</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="VirtualGamePad" name="virtualGamePad" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QWidget</extends>
   <header>imagelabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VirtualGamePad</class>
   <extends>QWidget</extends>
   <header>virtualgamepad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
